docs/.eleventy.js

```diff
@@ -0,0 +1,21 @@
+module.exports = function (eleventyConfig) {
+  // Copy assets to the output directory as-is
+  eleventyConfig.addPassthroughCopy("img");
+  eleventyConfig.addPassthroughCopy("css");
+  eleventyConfig.addPassthroughCopy("js");
+  eleventyConfig.addPassthroughCopy("vendor");
+  eleventyConfig.addPassthroughCopy("favicon.ico");
+
+  return {
+    // Control which files Eleventy will process
+    // e.g.: *.md, *.njk, *.html, *.liquid
+    templateFormats: ["html", "njk", "liquid", "md"],
+    // Path prefix for URLs
+    pathPrefix: "/",
+    // Config for input/output/data/etc. directories
+    dir: {
+      input: ".",
+      output: "_site",
```

Comment:
`eleventy --serve` just uses browser-sync internally (you can add a browser sync config to the `docs` folder even, I think). If we wanted it to be served as part of that, it would need to be included in the `docs` folder, which ultimately means it would be published along with the rest of the site.

I think we could do that, but it feels like we probably don't want to publish coverage as part of the website. Seems like we should just leave it for now.

Reply:
Agree that we don't want to publish coverage on the docs site.  That's a use case for codecov.io (e.g.).

See the `.bs-config` file for how I currently do it, adding:

```
    routes: {
      "/coverage": "./coverage/lcov-report",
    },
```

so that going to `http://localhost:8080/coverage` works.  None of that would show up at deploy time, unless I'm missing something.

Reply:
I did add a separate set of scripts for _only_ the `docs` folder, specifically for working on the docs site.

If you want to work on the docs site locally:
```sh
cd docs && npm run start
```

The browser sync config for the outer directory should still work as-is, but it won't work for the docs site correctly since it'll try to serve the raw HTML template files.